homeassistant/components/airnow/config_flow.py

```diff
@@ -94,12 +95,49 @@ async def async_step_user(self, user_input=None):
                     vol.Optional(
                         CONF_LONGITUDE, default=self.hass.config.longitude
                     ): cv.longitude,
-                    vol.Optional(CONF_RADIUS, default=150): int,
+                    vol.Optional(CONF_RADIUS, default=150): vol.All(
+                        int, vol.Range(min=5)
+                    ),
                 }
             ),
             errors=errors,
         )
 
+    @staticmethod
+    @core.callback
+    def async_get_options_flow(
+        config_entry: config_entries.ConfigEntry,
+    ) -> config_entries.OptionsFlow:
+        """Return the options flow."""
+        return OptionsFlowHandler(config_entry)
+
+
+class OptionsFlowHandler(config_entries.OptionsFlowWithConfigEntry):
+    """Handle an options flow for AirNow."""
+
+    async def async_step_init(self, user_input):
```

Comment:
```suggestion
    async def async_step_init(self, user_input: dict[str,Any] | None = None) -> FlowResult:
```
Can we do some proper typing